content/python/concepts/random-module/terms/uniform/uniform.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'uniform()'
+Description: 'Returns a pseudo-random floating-point number between two given numbers, inclusive.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.uniform()` method takes two numbers as arguments and returns a pseudo-random floating-point number between them, inclusive of the numbers provided.
+
+## Syntax
+
+```pseudo
+random.uniform(<low value>,<high value>)
+```
+
+Where `<low value>` is the number representing the lower bound of possible values and `<high value>` is a number representing the higher bound of possible values.
```

Comment:
According to [the official Python docs on `.uniform`](https://docs.python.org/3/library/random.html#random.uniform), the `high value` may not always be included because this function uses an equation, `a + (b-a) * random()`, that may discount it due to floating-point rounding.